app/(auth)/sign-up.tsx

```diff
@@ -1,129 +1,187 @@
+import { SafeAreaView, Text, TextInput, Pressable, View } from "react-native";
```

Comment:
Well done @AyaobaEstelle, 

I think you can also import { SafeAreaView } from "react-native-safe-area-context"; The one from react-native has some issues

Reply:
okay, ill work on it